vector/src/main/java/im/vector/adapters/VectorMessagesAdapterMediasHelper.java

```diff
@@ -243,6 +245,37 @@ void managePendingImageVideoDownload(final View convertView, final Event event,
                     thumbHeight = videoMessage.info.thumbnail_info.h;
                 }
             }
+        } else if (message instanceof StickerMessage) {
```

Comment:
This code should be useless now, no?
Plus I am wondering if the condition at https://github.com/vector-im/riot-android/pull/2119/files#diff-195abc8f7f6b2882511ac16bee261c64R200 is true before going to here.